### PR TITLE
Small correction about architectures near the beginning of the page

### PR DESCRIPTION
**Pull request by duckyondiscord**

### CODE REVIEW

**Comment by bsdjhb** [comment]
A good clarification, but I might reword it a bit further as:

```
The `amd64` indicates that the system in this example is running the port of FreeBSD to the 64-bit x86 architecture.
```

**Review by igalic — CHANGES_REQUESTED** [review]
note that while  ducky is a perfectly fine name, we don't accept contributions from emails that we can't actually email to, especially <x@users.noreply.github.com>

**Comment by duckyondiscord** [comment]
> note that while ducky is a perfectly fine name, we don't accept contributions from emails that we can't actually email to, especially [x@users.noreply.github.com](mailto:x@users.noreply.github.com)
Alright well you can email me at duccywastaken@pm.me then, I should add it to my github profile.

**Comment by duckyondiscord** [comment]
> A good clarification, but I might reword it a bit further as:
> 
> ```
> The `amd64` indicates that the system in this example is running the port of FreeBSD to the 64-bit x86 architecture.
> ```

I personally think that's unnecessarily complicating it a little. You can make that change if you want but I think simpler's better in this case.

File: documentation/content/en/books/handbook/basics/_index.adoc [hunk 1]
```diff
@@ -82,7 +82,7 @@ login:
 ....
 
 The first line contains some information about the system.
-The `amd64` indicates that the system in this example is running a 64-bit version of FreeBSD.
+The `amd64` indicates that the system in this example is running an x86 64-bit version of FreeBSD.
```
Comment (grahamperrin) [inline documentation/content/en/books/handbook/basics/_index.adoc:85]:
```suggestion
`amd64` indicates x86, 64-bit.
```
The [existing context](https://docs.freebsd.org/en/books/handbook/basics/#consoles):

<details>
<summary>screen shot</summary>

image

</details>

* a FreeBSD site
* a FreeBSD document.
* the _FreeBSD/_ part of `FreeBSD/amd64` needs no explanation. 

https://people.freebsd.org/~grahamperrin/style/writing/tips/ | be concise :-)

**Review by grahamperrin — CHANGES_REQUESTED** [review]

**Comment by grahamperrin** [comment]
https://github.com/freebsd/freebsd-doc/pull/281#issuecomment-1763549288

>> … the port of FreeBSD to …

+1

I get it, however …

… if I close my eyes, intentionally not reminding myself of what's already at page, I imagine <https://www.freebsd.org/about/> being a good/better place for something along the lines of: 

> FreeBSD and much of the complementary software in its ports collection is ported to ….

(Clumsily worded, but you get the idea.)

If a newcomer dives straight into the 'FreeBSD Basics' chapter with no prior knowledge of FreeBSD, no prior knowledge of the concept of porting: the reader might look away to <https://docs.freebsd.org/en/books/handbook/glossary/#_p> or <https://www.freebsd.org/ports/>  for a meaning of the word _port_.

HTH, happy to discuss in Matrix or IRC (to not derail things here).

**Comment by duckyondiscord** [comment]
Is there anything else I need to do here?

**Comment by bsdjhb** [comment]
Merged in 45ee577b18db51a4f64c3e64db1d3b409fc1d975.  Thanks!

**Pull request closed without merging**